### PR TITLE
Use inclusive language and correct references to default branches

### PR DESCRIPTION
**Pull request by mattlord**
Signed-off-by: Matt Lord <mattalord@gmail.com>

### CODE REVIEW

File: veps/vep-5.md [hunk 1]
```diff
@@ -4,20 +4,20 @@
 VEP: 5
 Title: Vitess Bugfixes
 Author: Andres Taylor
-Reviewers: Derek Perkins, Rafael Chacon, Harshit Gangal, Deepthi Sigireddi
+Reviewers: Derek Perkins, Rafael Chacon, Harshit Gangal, Deepthi Sigireddi, Matt Lord
 Status: Proposed
 Created: 2021-05-19
```
Comment (deepthi) [inline veps/vep-5.md]: - I think your name should go next to Andres as Author(s) (and not as a Reviewer)
- We should change Status to `Approved`
- Add another line: `Amended: <today's date>`

**Review by deepthi — COMMENTED** [review]
Thank you for noticing this. A few more changes are needed.

File: veps/vep-1.md [hunk 2]
```diff
@@ -85,7 +85,7 @@ The following is not subject to the Backwards and Forwards Compatibility Promise
 
 ## Support Lifecycle
 
-Because the release cycle includes a new release every 12 weeks, most fixes will only be pushed to the master branch. An exception to this process will be made for high severity bugs, in which case a fix will be available in the weekly release for all currently supported versions.
+Because the release cycle includes a new release every 12 weeks, most fixes will only be pushed to the main branch. An exception to this process will be made for high severity bugs, in which case a fix will be available in the weekly release for all currently supported versions.
 
 High severity bugs in this context include [CVEs](https://cve.mitre.org/), data corruption, wrong results or outage inducing issues. The Vitess team **MAY** decide to only backport bugs with higher user impact.
```
Comment (ajm188) [inline veps/vep-1.md:91]: Can you add a new section to the bottom of both of these docs to call out "notable edits"? I was thinking along the lines of the following:

```
## Edits

- <date>
    - <author>, <high-level summary of changes> (<link to PR>)
- <date2> (and so on)
```

Comment (deepthi) [reply veps/vep-1.md:91]: Oh I missed that _two_ docs changed. So yeah all new information should be added to both.

Comment (mattlord) [reply veps/vep-1.md:91]: Look OK now?

**Review by rafael — APPROVED** [review]
LGTM

**Review by systay — APPROVED** [review]
Thanks a lot for taking this on!

**Pull request merged**